Bukkit/src/main/java/net/minecrell/serverlistplus/bukkit/BukkitPlugin.java

```diff
@@ -117,6 +122,10 @@ public StatusRequest load(InetSocketAddress client) throws Exception {
 
     private LoadingCache<InetSocketAddress, StatusRequest> requestCache;
     private String requestCacheConf;
+    
+    private static boolean isPluginLoaded(String pluginName) {
+        return Bukkit.getPluginManager().getPlugin(pluginName) != null;
```

Comment:
Avoid using `Bukkit` in the main class, remove `static` and use `getServer()` instead.